scripts/remove_directories_with_few_files.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+import argparse
+import pathlib
+
+import colorama
+
+from arc_utilities.filesystem_utils import ask_to_remove_directories, count_files_recursive
+
+
+def main():
+    colorama.init(autoreset=True)
+    parser = argparse.ArgumentParser("finds directories smaller than a given size, asks for confirmation, then deletes")
+    parser.add_argument('root', type=pathlib.Path, help="root directory", nargs='+')
```

Comment:
Didn't know that there's a default pathlib.Path parser for argparse; could you add an example argument in the help string?

Reply:
there's no magic here, if you pass `Foo` to type, it just calls `Foo(the_arg_as_a_string)` so that's how this works

I'm not sure an example would be helpful, it's just a directory? can you suggest something?

Reply:
Ok, if there's no strange cases with treating spaces or trailing slashes in the argument then I'd be fine with not having an example (e.g. help="root directory specified for example \"~/Documents/my project\"")